tbconnect/tasks.py

```diff
@@ -132,3 +133,36 @@ def perform_etl():
         utils.sync_models_to_bigquery(
             settings.TBCONNECT_BQ_KEY_PATH, settings.TBCONNECT_BQ_DATASET, models
         )
+
+
+@shared_task
+def send_tbcheck_data_to_cci(data):
+    msisdn = data.get("msisdn")
+    profile = get_user_profile(msisdn)
+
+    if profile:
+        # update or append data with profile and gender
+        data.update({"province": profile.province, "gender": profile.gender})
+
+        # Send user data to cci
+        headers = {
+            "Authorization": f"Bearer {settings.CCI_AUT_TOKEN}",
+            "Content-Type": "application/json",
+        }
+        response = requests.post(url=settings.CCI_AUT_URL, headers=headers, data=data)
+
+        if response.status_code == 200:
+            return "CCI data submitted successfully"
+        response.raise_for_status()
+        return "CCI data Submission failed"
+    return "User profile not found"
```

Comment:
This message will never be seen because this is a task, maybe we should raise an exception here